CONTRIBUTING.md

```diff
@@ -4,7 +4,7 @@ The "RFC" (request for comments) process is intended to provide a consistent and
 
 ## When you need to follow this process
 
-You need to follow this process if you intend to make "substantial" changes to the specification or the RFC process itself. What constitutes a "substantial" change is evolving based on community norms and varies depending on what part of the specification you are proposing to change, but may include the following:
+You need to follow this process if you intend to make "substantial" changes to the specification or to the RFC process itself. What constitutes a "substantial" change is evolving based on community norms and varies depending on what part of the specification you are proposing to change, but may include the following:
```

Comment:
"to this or that" seems like acceptable English to me.  "to this or to that" is redundant.

Reply:
Done